test/vanilla/ignite-new.test.ts

```diff
@@ -287,54 +281,10 @@ describe("ignite new", () => {
     beforeAll(async () => {
       tempDir = tempy.directory({ prefix: "ignite-" })
 
-      result = await runIgnite(`new ${APP_NAME} --debug --packager=yarn --workflow=cng --yes`, {
```

Comment:
This diff is a little confusing, sorry; I noticed while updating this that this test for yarn was duplicated.